src/object3.jl

```diff
@@ -300,10 +300,14 @@ end
 # helper
 
 
-function rotate3d(x::RealU, y::RealU, z::RealU, ϕ::RealU, θ::RealU)
-    θ == 0 || throw(ArgumentError("θ ≂̸ 0 unsupported currently"))
-    (s, c) = sincos(ϕ)
-    return (c * x + s * y, -s * x + c * y, z)
+function rotate3d(x::RealU, y::RealU, z::RealU, ϕ::RealU, θ::RealU, ψ::RealU)
+    sinϕ, cosϕ = sincos(ψ)
+    sinθ, cosθ = sincos(θ)
+    sinψ, cosψ = sincos(-ϕ)
```

Comment:
Is the "mismatch" between \phi and \psi deliberate here?
If so, I'd like comment(s) added so that everyone (including our future selves) know why!

Reply:
That was just a quick hack since I was unsure which of the rotation axis ordering and sign you had in mind here.  

If we look at Wikipedia https://en.wikipedia.org/w/index.php?title=Rotation_matrix&section=9#In_three_dimensions we can see that the Rz is already incompatible with the code before (minus sign). Was that intended? I would actually just Rxyz or Rzyx as defined at wikipedia but I fear that the minus sign will break some relationship with the radon transform.